genie-proto/build.gradle

```diff
@@ -0,0 +1,81 @@
+apply plugin: 'com.google.protobuf'
```

Comment:
For consistency I've been going with `"` instead of `'` in the build.gradle files.

Reply:
Done